app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +33,20 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".game.drawing.DrawingOffline">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        <activity android:name=".game.drawing.DrawingOnline">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
```

Comment:
Please remove the intent-filter with MAIN and LAUNCHER from DrawingOffline and DrawingOnline, they cause the app to have multiple icons installed